OpenRA.Game/Server/Server.cs

```diff
@@ -20,106 +20,47 @@
 using OpenRA.Primitives;
 using OpenRA.Support;
 
-using XTimer = System.Timers.Timer;
-
 namespace OpenRA.Server
 {
-	public enum ServerState : int
+	public enum ServerState
 	{
-		WaitingPlayers = 1,
-		GameStarted = 2,
-		ShuttingDown = 3
+		WaitingPlayers, GameStarted, ShuttingDown
```

Comment:
"Why not" should is not a reason to change something that was well formatted into a one-liner. Nor for changing the default values.